.storybook/themes/news.scss

```diff
@@ -1367,6 +1367,40 @@
 					margin: 0,
 					padding: 0,
 				),
+				medium-promo: (
+					components: (
+						attribution: (
+							display: block,
+							margin-top: var(--global-spacing-4),
+						),
+						media-item: (
+							float: right,
+							max-width: 100px,
+						),
+						heading: (
+							font-size: var(--global-font-size-7),
+							font-weight: var(--global-font-weight-7),
+							line-height: var(--global-line-height-4),
+							margin-bottom: var(--global-spacing-4),
+						),
+						paragraph: (
+							display: inline,
+							font-size: var(--global-font-size-4),
+							line-height: var(--global-line-height-4),
```

Comment:
```suggestion
```
The default paragraph already uses the `body-font-size` and `body-font-line-height` which are `font-size-4` and the appropriate line-height for the font so these aren't needed unless you are overriding the Paragraph default.

Reply:
Exactly! Again, committed before I saw this feedback. These lines have been removed.